src/tag.spec-d.ts

```diff
@@ -6,33 +6,33 @@ import { tag } from './tag'
 test('tag function returns a tag when no callback is provided', () => {
   const value = tag()
 
-  expectTypeOf(value).toMatchTypeOf<QueryTag>()
+  expectTypeOf(value).toExtend<QueryTag>()
```

Comment:
This is because of the vitest deprecation yeah? What's the other option? Extends seems like the looser option.

Reply:
correct. Type hints for deprecated `toMatchTypeOf` suggested `toExtend`, didn't try too hard to find a better assertion 🤷